myapp/db/migrate/20221221025136_change_tasks_title_and_description_null_false.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeTasksTitleAndDescriptionNullFalse < ActiveRecord::Migration[6.0]
+  def up
+    change_column :tasks, :title, :string, null: false
+    change_column :tasks, :description, :string, null: false
```

Comment:
I haven't set restrictions to other columns as I think those are to be implemented in upcoming steps (I only display `title` and `description` on my view now)